cmd/kogito/command/message/trusty_ui.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package message
+
+import "fmt"
+
+var (
+	// TrustyUIErrCreating ...
+	TrustyUIErrCreating = fmt.Sprintf(serviceErrCreating, "Trusty UI", "%s")
+	// TrustyUISuccessfulInstalled ...
+	TrustyUISuccessfulInstalled = fmt.Sprintf(serviceSuccessfulInstalled, "Trusty UI", "%s")
+	// TrustyUICheckStatus ...
+	TrustyUICheckStatus = fmt.Sprintf(serviceCheckStatus, "kogitotrustyui", "%s", "%s")
+	// TrustyUINotInstalledNoKogitoOperator ...
+	TrustyUINotInstalledNoKogitoOperator = fmt.Sprintf(serviceNotInstalledNoKogitoOperator, "Trusty UI", "trusty-ui")
```

Comment:
NotInstalledNo?
what this "No" stands for?

Reply:
Hi @spolti , good question. tbh I followed the interface 
```
type serviceInfoMessages struct {
	errCreating                  string
	installed                    string
	checkStatus                  string
	notInstalledNoKogitoOperator string
}
```

iiuc this should be the message when the operator is not deployed (yet). I see that the `No` string is literally everywhere (`serviceNotInstalledNoKogitoOperator`, `JobsServiceNotInstalledNoKogitoOperator`, ...).  
@ricardozanini `git blame` suggests me to ask to you

Reply:
"No" stands for "Kogito Operator not installed/not ready". You can refactor to a better name if it fits.